integ-test/src/test/java/org/opensearch/sql/sql/HighlightFunctionIT.java

```diff
@@ -29,6 +29,40 @@ public void single_highlight_test() {
     assertEquals(1, response.getInt("total"));
   }
 
+  @Test
+  public void highlight_optional_arguments_test() {
+    String query = "SELECT highlight('Tags', pre_tags='<mark>', post_tags='</mark>') " +
+        "FROM %s WHERE match(Tags, 'yeast') LIMIT 1";
+    JSONObject response = executeJdbcRequest(String.format(query, TestsConstants.TEST_INDEX_BEER));
+    verifySchema(response, schema("highlight('Tags', pre_tags='<mark>', post_tags='</mark>')",
+            null, "nested"));
+
+    assertEquals(1, response.getInt("total"));
+
+    assertEquals("alcohol-level <mark>yeast</mark> home-brew champagne",
+        response.getJSONArray("datarows").getJSONArray(0).getJSONArray(0).getString(0));
+  }
+
+  @Test
+  public void highlight_multiple_optional_arguments_test() {
+    String query = "SELECT highlight(Title), highlight(Body, pre_tags='<mark style=\\\"background-color: " +
+        "green;\\\">', post_tags='</mark>') FROM %s WHERE multi_match([Title, Body], 'IPA') LIMIT 1";
+    JSONObject response = executeJdbcRequest(String.format(query, TestsConstants.TEST_INDEX_BEER));
+
+    verifySchema(response, schema("highlight(Title)", null, "nested"),
+        schema("highlight(Body, pre_tags='<mark style=\"background-color: green;\">', " +
+                "post_tags='</mark>')", null, "nested"));
+
+    assertEquals(1, response.getInt("total"));
+
+    assertEquals("What are the differences between an <em>IPA</em> and its variants?",
```

Comment:
Is it possible to use `verifyDataRows` in these tests? 

I think there's enough repeated code here to create a matcher that simplifies these tests.